app/controllers/users.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const bcrypt = require('bcrypt');
+const errors = require('../errors');
+const userService = require('../services/users');
+const sessionManager = require('./../services/sessionManager');
+const isAlphaNumeric = require('validate.io-alphanumeric');
```

Comment:
Beware when introducing dependencies for anything , the repository of this particular module has only 2 stars and the latest commit is from 2 years ago so I wouldnt use it , especially for a feature as simple as checking wether its alphanumeric. Change this validation to a regular expression match.